components/api/api-modules-javascript/src/main/resources/META-INF/dirigible/modules/src/http/index.ts

```diff
@@ -1,9 +1,9 @@
 export * as client from "./client";
-export * as asyncClient from "./client-async";
+export {HttpAsyncClient as asyncClient} from "./client-async";
 export * as rs from "./rs";
 export * as upload from "./upload";
-export * as request from "./request";
-export * as response from "./response";
-export * as session from "./session";
+export {Request as request} from "./request";
+export {Response as response} from "./response";
+export {Session as session} from "./session";
 export * as decorators from "./decorators";
 export * from "./decorators";
```

Comment:
Are these changes needed?  Isn't `export * ...` already fine?

Reply:
They are needed. Without them the importing process breaks, resulting in errors such as: <h3>*"Property 'isCommitted' does not exist on type 'typeof import("/<path_to_folder>/dirigible/components/api/api-modules-javascript/src/main/resources/META-INF/dirigible/modules/src/http/response")'."*</h3>

They are probably no longer considered the same class as the original.